oak_runtime/introspection_browser_client/package.json

```
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "dev": "npm ci && npx webpack-dev-server --env.NODE_ENV=development",
+    "dev": "./build \"development\"",
```

This should work I think?

```suggestion
    "dev": "./build development",
```

---

Does yeah, probably nicer this way :))